subprojects/build-option/src/main/java/org/gradle/internal/buildoption/AbstractBuildOption.java

```diff
@@ -0,0 +1,50 @@
+/*
+ * Copyright 2017 the original author or authors.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.gradle.internal.buildoption;
+
+/**
+ * Provides a basic infrastructure for build option implementations.
+ *
+ * @since 4.2
```

Comment:
⭕️  should adjust all these to 4.3

Reply:
Will do that though I am wondering if it's worth it as it is an internal API.

Reply:
done